src/checks.rs

```diff
@@ -13,6 +13,10 @@ trait Check {
     fn run(&mut self, line: &LineEntry) -> Option<Warning>;
 }
 
+trait Message {
+    fn message(&self, key: &str) -> String;
```

Comment:
Not all checks may have keys. For example: 
https://github.com/mgrachev/dotenv-linter/blob/master/src/checks/leading_character.rs#L11
https://github.com/mgrachev/dotenv-linter/blob/master/src/checks/space_character.rs#L11